easybuild/easyconfigs/k/KITE-Tools/KITE-Tools-1.1-foss-2022a.eb

```diff
@@ -0,0 +1,49 @@
+# This is the tool-box for pre- and post-processinf of KITE files
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeMake'
+
+name = 'KITE-Tools'
+version = '1.1'
+
+homepage = 'https://github.com/quantum-kite/kite'
+description = """KITE is an open-source Python/C++ software suite for efficient real-space tight-binding (TB)
+simulations of electronic structure and bulk quantum transport properties of disordered systems scalable to
+multi billions of atomic orbitals.
+This is the KITE-Tools part of the program."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['https://github.com/quantum-kite/kite/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['c1654cd63192b004b95cf50913a43e1c449760af04730f6ca91c44089ff4d5e2']
+
+builddependencies = [
+    ('binutils', '2.38'),
```

Comment:
No binutils for toolchains other than GCCcore

Reply:
Thanks! I changed that.